### PR TITLE
To Do list: interactive lists

### PR DESCRIPTION
**Pull request by Moses-chibuike**
Added a new JavaScript file 
Added event listener to the checkbox 
Used HTML, CSS and JavaScript best practices
Update Readme

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable quotes */
```
Comment (cynthiainga) [inline src/index.js]: - [ ] Please kindly do not disable the linters, fix linters help us to write a clean and professional codebase 🙃 😉

File: src/mark.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes */
```
Comment (cynthiainga) [inline src/mark.js]: - [ ] Please kindly do not disable the linters, fix linters help us to write a clean and professional codebase 🙃 😉

File: src/todo.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
+/* eslint-disable quotes */
```
Comment (cynthiainga) [inline src/todo.js]: - [ ] Please kindly do not disable the linters, fix linters help us to write a clean and professional codebase 🙃 😉

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Petrus
```
Comment (cynthiainga) [inline LICENSE]: - [OPTIONAL] I think it could be better if your LICENSE file is updated 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Moses-chibuike, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Correct GitHub flow ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Moses-chibuike, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ]  Correct GitHub flow ✔️
- [ ]  Descriptive PR ✔️
- [ ]  Good commit history ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**